lib/requests.js

```diff
@@ -27,14 +29,17 @@ function send(config, options, callback) {
   var url = config.getBaseURL() + path + "?" + queryString;
 
   var params = {
-  	url: url,
-  	proxy: config.proxy,
-  	timeout: config.timeout
+    url: url,
+    proxy: config.proxy,
+    timeout: config.timeout,
+    headers: {
+      'x-pusher-library': 'pusher-http-node ' + packageJson.version
+    }
   };
 
   if(body) {
-  	params.headers = {'content-type': 'application/json'};
-  	params.body = body;
+    params.headers['content-type'] = 'application/json';
+    params.body = body;
```

Comment:
Also note the whitespace inconsistencies, I think this project defaults to 4 white spaces per tab, your editor is set to 2. In some parts of the world this is considered criminal!